websites/M/Mastodon/presence.ts

```diff
@@ -0,0 +1,92 @@
+const presence = new Presence({
+		clientId: "1107321415243415574",
+	});
+
+
+// checks if user is logged from textarea
+function checkPositionAccountAvatar() {
+	if (document.querySelectorAll(".autosuggest-textarea__textarea").length < 1)
+		return 1;
+	return 2;
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/jlQv2if.png",
+	};
+	
+	if (document.location.pathname === "/home") {
```

Comment:
`document.location` is used many times in this presence. Destructure the properties to simplify the code a bit:
```ts
const { pathname } = document.location;
```